BraveWallet/Crypto/Onboarding/PasswordValidation.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+struct PasswordValidation {
+  /// Whether or not a password is valid to use when creating or restoring a wallet
+  ///
+  /// Currently a strong password is defined by having at least 7 characters, and including at least
```

Comment:
Let's change the wording here to "minimum". 7 characters is not a strong password by any security definition.